### ISSUE TITLE
Honorspy reset again in EU in tuesday even tho it should be wednesday (happened last week as well).

### ISSUE THREAD

**Issue by Finrodi** [kakysha/HonorSpy#112]
title.

**Comment by Goldpaw** [comment]
Resets on Wednesday for me. Have you got the latest version? And if you type `/run print(GetCurrentRegion())` and hit Enter within the game, what is the output of that? If it's 1, the problem is in your game client, not the addon.

**Comment by Finrodi** [comment]
Newest version and it says 1. However I'm not the only one with the problem, got multiple people in my guild with the same problem. Any idea how to fix this?

**Comment by kakysha** [comment]
How many of you got this problem? What game client do you use? Where did you get it? What in-game language you have? What servers do you play?
If you are from EU and playing on EU servers, `GetCurrentRegion()` should return 3 instead of 1, as @goldpaw correctly said.
There is a note on https://wow.gamepedia.com/API_GetCurrentRegion saying that
```
This function is unreliable for players who switch between regions. For example, if a player installs the US client and then logs into EU servers, this function will incorrectly report that they are still in the US region.
```
Is it the case for you?

**Comment by Finrodi** [comment]
Asked in teamspeak 5 out of 7 have the problem. All 5 have ingame region "1" the two without the problem "3".

Not sure what you mean about game client, I'm using the "standard" one, never played on US servers. Ingame language is English, playing on Gehennas EU.

**Comment by Finrodi** [comment]
Also this problem started happening roughly 2 weeks ago. Never had that issue during wpvp for sure and start of AV.

**Comment by kakysha** [comment]
That sounds like a very strange game client behavior, and has nothing to do with the addon actually. The game just reports wrong region for you. If this is the case for the majority of players, I probably need to hack the way around this game client bug. Probably `GetRealmID()`?

**Comment by Finrodi** [comment]
Its very strange especially since it started happening for all of us I believe the Tuesday 9 days ago. Would be awesome if you somehow could find a fix for that.

**Comment by AterAter** [comment]
I would like to bump this. Same happened today as it has been doing for at least three weeks now. Checked with my guild last week, the addon reset on Tuesday instead of Wednesday for about half the guild. Checked with the few that are online today and yes it rested for myself and a few of the people who where online this week too.

EU server Gehennas, english clients. Please give us an option to set reset day from the addons config ingame.

**Comment by AterAter** [comment]
Worth mention is that the same thing happens both at my computer at home and my computer at work.

**Comment by Finrodi** [comment]
Can confirm that it happened again today.

**Comment by 3n8** [comment]
This has happened for me foure weeks in a row.
I googled for this https://eu.battle.net/support/en/article/12022
And checked my clients launcher.db The region was wrong. I set it to enGB
I then checked my config.wtf region, locale and audolocale they were also set to US. I set it to enGB
Despite that i play on EU exclusively.

I then logged in and my honorspy data started populating. I was happy
I then restarted the game and the game had set the locale but not region in my config.wtf back to:
SET textLocale "enUS"
SET audioLocale "enUS"

I have repeated these steps a few times and the game keeps setting the locale and audiolocale in config.wtf to enUS despite that  i don't play there. But that might not be a bug.
but: /run print(GetCurrentRegion()) reports region 3 now.
Since i have no faith in blizzard at all, Any chance you would add a setting for us to pick region?
